understanding/language-of-page.html

```diff
@@ -55,82 +53,68 @@
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Assistive technology can determine the language of a page.</dd>
+                  <dt>目標</dt>
+                  <dd>支援技術はページの言語を決定できる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Indicate the predominant language on a page.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>ページの主な言語を示す。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People using assistive technology get information in the correct language.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>支援技術を使用する人々は、正しい言語で情報を受け取る。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
+               <h2>意図</h2>
                <p>
-                  The intent of this Success Criterion is to ensure that content developers provide
-                  information in the Web page that user agents need to present text and other linguistic
-                  content correctly. Both assistive technologies and conventional user agents can render
-                  text more accurately when the language of the Web page is identified. Screen readers
-                  can load the correct pronunciation rules. Visual browsers can display characters and
-                  scripts correctly. Media players can show captions correctly. As a result, users with
-                  disabilities will be better able to understand the content.
+                  この達成基準の意図は、ユーザエージェントがテキスト及びその他の形式の言語学上の言語によるコンテンツを正しく提示するために必要なウェブページの情報を、コンテンツ制作者が提供することを保証することである。支援技術と従来のユーザエージェントはともに、ウェブページの言語が識別されるとき、テキストをより正確に描画することができる。スクリーンリーダーは、正しい発音規則を読み込むことができる。ビジュアルブラウザは、文字や書体を正しく表示することができる。メディアプレーヤーは、キャプションを正しく表示できる。結果として、障害のある利用者がコンテンツをより理解できるようになる。
                   
                   
                </p>
-               <p>The default human language of the Web page is the default text-processing language
-                  as discussed in 
-                  <a href="https://www.w3.org/International/techniques/authoring-html#gslang">Internationalization Best Practices: Specifying Language in XHTML &amp; HTML Content</a>. When a Web page uses several languages, the default text-processing language is
-                  the language which is used most. (If several languages are used equally, the first
-                  language used should be chosen as the default human language.) 
+               <p><a href="https://www.w3.org/International/techniques/authoring-html#gslang">Internationalization Best Practices: Specifying Language in XHTML &amp; HTML Content</a> で述べられているように、ウェブページのデフォルトの自然言語が、デフォルトのテキスト処理言語となる。ウェブページがいくつかの言語を使用している時、デフォルトのテキスト処理言語は、最も使われている言語である。(複数の言語が同じ割合で使われている場合、最初に使われている言語をデフォルトの自然言語として選択する必要がある。)
```

Comment:
should be を「選択する必要がある」としているのは強すぎるかもしれません。「選択するべき」くらいでどうでしょうか。

```suggestion
               <p><a href="https://www.w3.org/International/techniques/authoring-html#gslang">Internationalization Best Practices: Specifying Language in XHTML &amp; HTML Content</a> で述べられているように、ウェブページのデフォルトの自然言語が、デフォルトのテキスト処理言語となる。ウェブページがいくつかの言語を使用している時、デフォルトのテキスト処理言語は、最も使われている言語である。(複数の言語が同じ割合で使われている場合、最初に使われている言語をデフォルトの自然言語として選択するべきである。)
```